packages/dds/matrix/src/test/time/sharedMatrix.spec.ts

```diff
@@ -0,0 +1,134 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
Why did we disable the no-non-null-assertion linter rule? I don't see the use of `foo!` in this test suite?

Reply:
(If we need it, we should leave a comment explaining why)

Reply:
You are right. The `foo!` is no longer exist here. Remove this disable rule here.